x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -652,4 +654,71 @@ export function trainedModelsRoutes({
         }
       })
     );
+
+  /**
+   * @apiGroup TrainedModels
+   *
+   * @api {post} /internal/ml/trained_models/model_downloads Gets available models for download
+   * @apiName GetTrainedModelDownloadList
+   * @apiDescription Gets available models for download.
```

Comment:
Can you mention it looks for the OS and arch in the description.

Reply:
Updated in https://github.com/elastic/kibana/pull/167000/commits/3cbb7fd3b7647237aedca990ce2755d3570493ac